pkgs/applications/window-managers/spectrwm/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation {
   pname = "spectrwm";
-  version = "unstable-2023-05-07";
+  version = "3.5.1";
 
   src = fetchFromGitHub {
     owner = "conformal";
     repo = "spectrwm";
-    rev = "06e3733175969c307a6fd47240a7a37b29d60513";
-    sha256 = "QcEwFg9QTi+cCl2JghKOzEZ19LP/ZFMbZJAMJ0BLH9M=";
+    rev = "SPECTRWM_3_5_1";
+    sha256 = "Nlzo35OsNqFbR6nl3nnGXDWmwc8JlP4tyDuIGtKTnIY=";
```

Comment:
```suggestion
    hash = "sha256-Nlzo35OsNqFbR6nl3nnGXDWmwc8JlP4tyDuIGtKTnIY=";
```

Reply:
Thank you for your improvement suggestions @wegank!